web/src/lib/utils/helpers.ts

```diff
@@ -111,22 +111,41 @@ export const scrollToTop = (props?: { smooth?: boolean }): void => {
     : window.scrollTo(0, 0);
 };
 
+export const NAVBAR_WRAPPER_MOBILE_ID = "NAVBAR_WRAPPER_MOBILE_ID";
+export const NAVBAR_WRAPPER_DESKTOP_ID = "NAVBAR_WRAPPER_DESKTOP_ID";
+
+const getNavBarHeight = (): number | undefined => {
+  const navbarMobile = document.querySelector(`#${NAVBAR_WRAPPER_MOBILE_ID}`);
+  const navbarDesktop = document.querySelector(`#${NAVBAR_WRAPPER_DESKTOP_ID}`);
+
+  if (navbarDesktop && navbarMobile) {
+    const heightDesktop = Number.parseInt(getComputedStyle(navbarDesktop).height);
+    const heightMobile = Number.parseInt(getComputedStyle(navbarMobile).height);
+    if (!Number.isNaN(heightDesktop)) {
+      return heightDesktop;
+    } else if (!Number.isNaN(heightMobile)) {
+      return heightMobile;
+    }
+  }
+  return undefined;
+};
+
 export const scrollToTopOfElement = (
   element: HTMLDivElement | null,
-  { isDesktop, waitTime = 100 }: { isDesktop: boolean; waitTime?: number }
+  { waitTime = 100 }: { waitTime?: number }
```

Comment:
[dust] I think at this point we can just promote `waitTime` to be a full parameter, no? This is a very Pythonic approach to function parameters; not sure why we implemented it this way in our codebase.